hello-world/tests/helpers/pyspark_test.py

```diff
@@ -0,0 +1,28 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
```

Comment:
I can see the benefits of just using the standard library `unittest`. Would you consider using `pytest` instead? Having said that, I am quite keen on using the standard library where possible.

Reply:
Yeah since this is a public example I was going to just use the standard library